portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Listing/Listing.tsx

```diff
@@ -0,0 +1,1115 @@
+        promise
+            .then((response) => {
+                if (response.status === 200 || response.status === 201) {
+                    setLoading(false);              
+                    APIMAlert.success(intl.formatMessage({
+                        id: 'Policy.Deploy.Successful',
+                        defaultMessage: 'Policy deployed successfully',
+                    }));    
+                    /**
+                     * If successful, add to the state rather than getting from backend.
+                     */
+                    addLabelsToPolicy(
+                        gatewayPolicyMappingId, 
+                        deployingGateways
+                    );                                         
+                }
+                else {
+                    APIMAlert.error(intl.formatMessage({
+                        id: 'Error.Deploy.Policy',
+                        defaultMessage: 'Error occurred while deploying the policy',
+                    }));
+                }                
+            })
+            .catch((/* error */) => {
+                // console.error(error);
+                APIMAlert.error(intl.formatMessage({
+                    id: 'Error.Deploy.Policy',
+                    defaultMessage: 'Error occurred while deploying the policy',
+                }));             
+            })         
+        setLoading(false);
+    }
+
+    const getAllDepoloyedGateways = () => {
+        const allGateways = policies.map((policy) => 
+            getAppliedGatewayLabelsById(policy.id)).reduce((acc, val) => acc.concat(val), []);
+        return allGateways;
+    }
+
+    /**
+     * Function to delete a policy mapping
+     * @param {string} gatewayPolicyMappingId - Policy Identifier.
+     */
+    const deletePolicy = (gatewayPolicyMappingId: string) => {
+        setIsDeleteDialogOpen(false);
+        setLoading(true);
+        /**
+         * call the backend API and handle the response
+         */
+        const promise = API.deleteGatewayPolicyByPolicyId(gatewayPolicyMappingId);
+        promise
+            .then(() => {
+                /**
+                 * If successful, remove from the state rather than getting from backend.
+                 */
+                const updatedPolicies = policies.filter((policy) => policy.id !== gatewayPolicyMappingId);
+                setPolicies(updatedPolicies);
+                // Remove from the selected gateways list as well
+                setSelectedGateways(selectedGateways.filter((gateway) => gateway.id !== gatewayPolicyMappingId));
+                APIMAlert.success(intl.formatMessage({
+                    id: 'Policy.Delete.Successful',
+                    defaultMessage: 'Policy deleted successfully',
+                }));
+                setLoading(false);
+            })
+            .catch((/* error */) => {
+                // console.error(error);
```

Comment:
```suggestion
                
```

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)